news/2024/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: "The Followpoint: D I O, the Mastermind Behind osu! Tournaments"
+date: 2024-09-30 17:00:00 +0000
+series: The Followpoint
+---
+
+Hey there, do you like osu! tournaments? Then this interview read might just be for you!
+
+![](/wiki/shared/news/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments/banner.jpg)
+
+Welcome back to The Followpoint where we are back in business with another interview for you to enjoy! This time it's with [D I O](https://osu.ppy.sh/users/3958619), you've probably heard of him because of his outstanding contributions to many tournaments around the osu! tournament scene, so much in fact that it would probably take forever to list how many tournaments he's contributed to!
+
+Let's not waste any time and get on with the interview!
+
+---
+
+<style>
+    .news-chat-quote__avatar {
+        float: left;
+        width: 40px;
+        height: 40px;
+        border-radius: 50%;
+        margin-left: -50px;
+    }
+
+    .news-chat-quote__text-container {
+        margin-left: 50px;
+    }
+
+    .news-chat-quote__username {
+        font-weight: 600;
+        margin-bottom: 2px;
+    }
+
+    .news-chat-quote__colour-lvd {
+        color: #FFD1DC;
+    }
+
+    .news-chat-quote__colour-gmt {
+        color: #99EB47;
+    }
+</style>
+
+<div class="news-chat-quote__text-container">
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments/avatar-Utiba.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+Hello D I O, thanks for coming on! As we always do, let's start with an introduction. How would you introduce yourself to the people who don't know you?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3958619" style="background-image: url('/wiki/shared/news/2024-09-30-the-followpoint-d-i-o-the-mastermind-behind-osu-tournaments/avatar-D I O.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-gmt" href="https://osu.ppy.sh/users/3958619">D I O</a></p>
+
+Hi Utiba, thanks for having me. For those who don't know me, I am D I O, a GMT, member of Tournament Committee, OWC Caster and Playtester, host of the Perennial, head of tournament operations for the Roundtable, and community tournament staff since 2018. I joined osu! back in 2014.
```

Comment:
```suggestion
Hi Utiba, thanks for having me. For those who don't know me, I am D I O, a [GMT](/wiki/People/Global_Moderation_Team), member of [Tournament Committee](/wiki/People/Tournament_Committee), OWC caster and playtester, host of the [Perennial](https://osu.ppy.sh/home/news/2024-07-28-the-perennial-concludes), head of tournament operations for the [Roundtable](/wiki/Tournaments/The_Roundtable), and community tournament staff since 2018. I joined osu! back in 2014.
```

Reply:
Would be nice to have a link to the Perennial website (https://theperennial.net/) either here or in the question about the Perennial.

Reply:
Can do that